lasagne/layers/conv.py

```diff
@@ -117,23 +109,22 @@ class Conv1DLayer(Layer):
         An integer or a 1-element tuple specifying the stride of the
         convolution operation.
 
-    border_mode : str, one of 'valid', 'full', 'same'
-        A string indicating the convolution border mode.
+    pad : {0, (0,), 'full', 'same'} (default: 0)
```

Comment:
It seems a bit strange to me to have 0 here instead of something like 'int' or 'n'. Also, is it necessary to repeat the default from the signature? If so, it should probably be included for all arguments.